### DISCUSSION TITLE
Newbie is baffled

### DISCUSSION THREAD

**Discussion by Dan-Essin** [orbitalquark/textadept#249] [General]
Hello,
I need help with textadept in terminal mode. I am working on raspberry pis vis ssh so gui is out. I have read the manual and find it quite comprehensive when it comes to the gui.

I can't say the same about curses mode. There is virtually no documentation at all.
How does one exit?
How does one save a file?
Do the cursor keys navigate in the text or are there special key combinations?
etc.

By randomly poling alt-key combinations I have been able to switch buffers and get the find replace dialog but that's it.

It's really a shame that after putting in so much work to make a nice app that newcomers are left adrift like this. Perhaps there is some other page of documentation lurking somewhere, but I have searched and have not found it. I have tried looking as the source but lua.keys didn't help much. THe detailed bindings must be somewhere in the code but I didn't examine every file.

A bit of help would be appreciated

**Comment by orbitalquark** [comment]
Sorry for the trouble :( A list of keybindings is here: https://orbitalquark.github.io/textadept/api.html#textadept.keys

**Comment by orbitalquark** [comment]
I have committed a change to the FAQ that points to this list. https://github.com/orbitalquark/textadept/commit/9dadbb37d625b4ecac023450ae71e30d103d2a58. Hopefully this will help others coming to Textadept. Thanks for bringing it up.
